lucien-ios/RootViewController.swift

```diff
@@ -8,13 +8,14 @@
 
 import UIKit
 
-class RootViewController: UIViewController {
+final class RootViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
     }
 
     @IBAction func signInButtonPressed(_ sender: UIButton) {
-        view.backgroundColor = UIColor.black
+        let startMyCollectionViewController = StartMyCollectionViewController(nibName: "StartMyCollectionViewController", bundle: Bundle.main)
```

Comment:
You can omit `nibName` and `bundle` here. UIKit will by default search in the main bundle for a nib whose name matches the class'.